.github/workflows/bins.yaml

```diff
@@ -101,3 +101,9 @@ jobs:
       package: node-exporter
     secrets:
       token: ${{ secrets.HUB_JWT }}
+  tpm:
+    uses: ./.github/workflows/bin-package.yaml
```

Comment:
bin-package.yaml uses `ubunut-latest` for the build. This is mostly fine because all other packages are doing "static builds" so they don't dynamically link to libraries that are available on the base image. Unlike tpm which can't be build statically (or at least I couldn't figure out how) this means the tpm and tss need to be build against the same version of the base image libraries. 

The 0-initramfs builds on ubuntu-18.04 hence the tpm pacakge needs also to build against the same ubuntu version.

Hence i suggest that u make a copy of `bin-pacakge.yaml` (name it `bin-pacakge-18.04.yaml` or something) then change it to use the right ubuntu version, then use it in this line instead of this one.